src/tests/apps.cpp

```diff
@@ -55,6 +55,37 @@ void annotatedTextSentences(Ptr<Options> options, bool source) {
   }
 }
 
+void qualityEstimatorWords(const Ptr<Options> &options) {
+  ResponseOptions responseOptions;
+  responseOptions.qualityScores = true;
+  const Response response = translateFromStdin(options, responseOptions);
+
+  for (const auto &sentenceQualityEstimate : response.qualityScores) {
+    std::cout << "[SentenceBegin]\n";
+
+    for (const auto &wordByteRange : sentenceQualityEstimate.wordByteRanges) {
+      const string_view word(response.target.text.data() + wordByteRange.begin, wordByteRange.size());
+      std::cout << word << "\n";
+    }
+    std::cout << "[SentenceEnd]\n\n";
+  }
+}
+
+void qualityEstimatorScores(const Ptr<Options> &options) {
+  ResponseOptions responseOptions;
+  responseOptions.qualityScores = true;
+  const Response response = translateFromStdin(options, responseOptions);
+
+  for (const auto &sentenceQualityEstimate : response.qualityScores) {
+    std::cout << std::fixed << std::setprecision(3) << sentenceQualityEstimate.sentenceScore << "\n";
```

Comment:
I think `sentenceQualityEstimate.sentenceScore` is better than `sentenceQualityScore.sentenceScore`. I think something was changed to `Score` from `Estimate` to make consistent somewhere. This is still inconsistent is it not?

Feel free to give the naming some more thought, because it'll seep to bindings and JS sources.